README.md

```diff
@@ -15,17 +15,31 @@ A SPT-Aki mod to be used with SPT-Aki Server to allow the Coop Module to communi
 
 ## Summary
 
-This is the SERVER modification of [SPT-Aki](https://www.sp-tarkov.com/) to allow the [Coop Module](https://github.com/paulov-t/SIT.Core) to communicate with the SPT-Aki Server.
+This is the SERVER modification of [SPT-Aki](https://www.sp-tarkov.com/) to allow the [SIT Client mod](https://github.com/stayintarkov/StayInTarkov.Client) to communicate with the SPT-Aki Server.
 
-## How to use this Repo?
+**NOTE:** The server mod is useless without the SIT Client mod. See the repo (linked above) for instructions on installing it. 
 
-* Install SIT via the Launcher (or manually)
```

Comment:
I "install SIT" from being a server install step, to being referenced above on line 20, and made it specifically reference the client mod. "SIT" on it's own is unbelievably ambiguous given the ~6 some-odd repos in the SIT github org lol. The clientmod also isn't part of the server install itself.

I feel like y'all could use some definitive nomenclature to better differentiate references to "The SIT Client Mod", "The SIT SPT-Aki Server Mod", "The SPT-Aki Client", and "The SPT-Aki Server"

If you're a fan of acronyms within acronyms, you could just use SCM, SSASM, SAC, SAS. Or maybe STCM, for SIT Tarkov Client Mod, to better align with "SIT SPT-Aki Server Mod" (ie, `[Project Name] [Thing this part modifies] [Type of mod]`, in case one were to interpret that it was the "client mod for sit", instead of sit's client mod for tarkov. (is this too much grammatical analysis for a foss project based on a foss project based on a goofy realistic shootergame? ¯\_(ツ)_/¯))

Reply:
I get the point, but as long as we provide the details on what to install in which location and provide links, it should be clear enough.